.github/workflows/benchmark_harness.yaml

```diff
@@ -55,7 +55,9 @@ jobs:
       matrix:
         # Add macos-latest and/or windows-latest if relevant for this package.
         os: [ubuntu-latest]
-        sdk: [3.2, dev]
+        # Have to put `3.10.0` instead of `3.10` because of
```

Comment:
You can also put `'3.10'` which is what I did in another package. I think either pattern is fine.